src/website/views/components/tooltip/_examples.pug

```diff
@@ -24,6 +24,30 @@ p.-text
 
       <script>chi.tooltip(document.getElementById('tooltip-1'));</script>
 
+h3 Light
+p.-text
+  | To render a light color tooltip, apply the <code>data-tooltip-color="light"</code> attribute to Tooltip trigger element.
```

Comment:
Can we slightly adjust this to ensure we communicate these are for dark backgrounds?

```suggestion
  | Use light tooltips for rendering tooltips on dark backgrounds. To use, apply the <code>data-tooltip-color="light"</code> attribute to the Tooltip trigger element.
```

Reply:
Makes sense